step.yml

```diff
@@ -0,0 +1,28 @@
+title: Activate Remote Gradle Cache
+summary: TODO
+description: |
+  TODO
+
+website: https://github.com/bitrise-steplib/bitrise-step-activate-gradle-remote-cache
+source_code_url: https://github.com/bitrise-steplib/bitrise-step-activate-gradle-remote-cache
+support_url: https://github.com/bitrise-steplib/bitrise-step-activate-gradle-remote-cache
+
+type_tags:
+- utility
```

Comment:
is this utility only?

Reply:
We don't really have a better-fitting category at the moment. The `utility` category is a catch-all for various steps, we might revisit this in the future.